lib/components/Auth/AuthService.ts

```diff
@@ -72,6 +78,38 @@ export function comparePassword(unencryptedPassword: string, encryptedPassword:
 
 
 
+export function setJWTTokenCookie(tokenStorageObj, res) {
+    const token = generateAuthToken({
+        user_id: tokenStorageObj.user_id
+    });
+
+    res.cookie('jwt', token, {
```

Comment:
Creates and sets the token, accessible by http only and has an expiration of 7 days